app/metrics/MetricsKeys.scala

```diff
@@ -54,6 +54,10 @@ object MetricsKeys {
     val Update = "push-notifications-backend-update"
   }
 
+  object UpscanInitiateBacked {
```

Comment:
UpscanInitiateBacked should be UpscanInitiateBackend, can you also please fix PushNotificationsBacked...should be PushNotificationsBackend